docs/DeveloperGuide.md

```diff
@@ -1,6 +1,6 @@
-# AddressBook Level 4 - Developer Guide
```

Comment:
Remember to update the Design when you make changes to the code base.
You don't want to push the documentations thing in the end right?